src/server/game/Entities/Unit/Unit.h

```diff
@@ -1774,6 +1774,7 @@ class FC_GAME_API Unit : public WorldObject
         TransportBase* GetDirectTransport() const;
 
         bool m_ControlledByPlayer;
+        static Creature* GetCreature(WorldObject& object, uint64 guid);
```

Comment:
@Asmadeuxx I have only one question, where is the implementation of this method?